### PR TITLE
New Beach Ruin: Knight's Rest

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Adds the Knight's Rest Ruin to the beach spawning pool. 
Knight's Rest is a small vaguely castle shaped object with a grave holding a deceased knight inside of it. Those who are not inclined to grave robbing will find basic growing equipment, torches, and some clothing. Those who do rob graves will find the dead knight's KICKASS SWORD. 
It's more of a setpiece than an adventure but having more beach ruins isn't hurting yet.

dreamseeker_7l4t7gggot
dreamseeker_iEdXdgwqMk
dreamseeker_PJJj9xgMrm
dreamseeker_7USXwcSxkc

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game
More beach ruin variety.

## Changelog

:cl:
add: Roamers of beach worlds have reported a small fortification, surrounded by a serene grass field. Perhaps there's something to investigate there.
/:cl:

### CODE REVIEW

File: _maps/map_catalogue.txt [hunk 1]
```diff
@@ -473,6 +473,9 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		Size = (x = 18)(y = 14)(z = 1)
 		Tags = "Medium Combot Challenge", "Medium Loot", "Liveable"
 
+		File Name = "_maps\RandomRuins\BeachRuins\beach_knights_rest.dmm"
+		Size = (x = 48)(y = 48)(z =1)
+		tags = "No Combat", "Medium Loot"
```
Comment (Latentish) [inline _maps/map_catalogue.txt]: Needs an atmosphere value

**Review by Latentish — DISMISSED** [review]
This needs a small change to the catalogue entry.
Otherwise this can be tm'd while it is passing.

**Comment by Latentish** [comment]
image
Btw you're using a weird grass as well. Ignoring the clowns, they lead to space. You want this one.
image

**Review by ZephyrTFA — APPROVED** [review]

**Review by Latentish — DISMISSED** [review]
> image Btw you're using a weird grass as well. Ignoring the clowns, they lead to space. You want this one. image

This along with every other ruin using these tiles, need to have proper beach planet asteroid tile subtypes.  Or an otherwise properly set up baseturf. I mentioned this in this prior post, but I should have made it as a request change.

**Review by spockye — APPROVED** [review]

**Pull request merged**